Source/Frontend/UI/Components/Engine Config/RTC_GeneralParameters_Form.cs

```diff
@@ -31,31 +31,10 @@ private void RTC_GeneralParameters_Form_Load(object sender, EventArgs e)
 
         private void cbBlastRadius_SelectedIndexChanged(object sender, EventArgs e)
         {
-            switch (cbBlastRadius.SelectedItem.ToString())
+            var algorithm = BlastRadius.GetFromString(cbBlastRadius.SelectedItem.ToString());
+            if (algorithm != null)
             {
-                case "SPREAD":
```

Comment:
I'll want to test all of these algorithms at least once. Ironically, I haven't used the corruptor too much, so it will be a learning experience for me to work through these scenarios and make sure they are all working.

I'll leave this change as a draft until I can validate unless you feel fine with potential regressions here.